sass/_variables.scss

```diff
@@ -4,9 +4,9 @@
 
 $base-font: 'Roboto', helvetica, sans-serif !default;
 
-$font-size-base: 12px !default;
-$font-size-button: 11px !default;
-$font-size-toolbar: 14px !default;
+$font-size-base: 1em !default;
+$font-size-button: 1em !default;
+$font-size-toolbar: 1em !default;
```

Comment:
@ashkanx Should these variable be `1.2em`, `1.1em` and `1.4em` respectively? or all of them have been changed to `10px`in the new design?

Reply:
I'm not sure yet. still waiting for UI team